examples/cymbal-bank/4-appfactory/cymbal-bank/main.tf

```diff
@@ -52,7 +52,7 @@ module "components" {
   for_each = tomap({
     for app_service in local.expanded_app_services : "${app_service.app_name}.${app_service.service_name}" => app_service
   })
-  source = "../../modules/app-group-baseline"
+  source = "../../../../4-appfactory/modules/app-group-baseline"
```

Comment:
should we change to terraform registry path?

Reply:
Hi @amandakarina! The repo HEAD and PRs need to point to "their" version of the modules. For our module repositories this is done automatically for the lint and test by https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/infra/module-swapper. I believe it should work automatically for this repo (as it follows the standard `terraform-google-` naming convention), however we should double-check given the non-standard layout.

If we do use the registry, we should also add a version constraint.